docs/source/deployment/apache.conf

```diff
@@ -9,14 +9,12 @@
     Alias /static/ /opt/ganeti_webmgr/collected_static/
 
     <Directory /opt/ganeti_webmgr/collected_static>
-        Order deny,allow
-        Allow from all
+        Require all granted
```

Comment:
This will break on 2.2 though. You need to have this set depending on the ``default['apache']['version']`` attribute.